### PR TITLE
Enhance verbose mode

### PR DESCRIPTION
**Pull request by PLangowski**

### CODE REVIEW

File: include/dts-functions.sh [hunk 1]
```diff
@@ -1526,10 +1526,10 @@ main_menu_options(){
           # dts scripting should get some LOGLEVEL and maybe dumping working
           # logs to file
           export DEPLOY_REPORT="false"
-          wait_for_network_connection && ${CMD_DASHARO_HCL_REPORT} && LOGS_SENT="1"
+          wait_for_network_connection &&  ${CMD_DASHARO_HCL_REPORT} && LOGS_SENT="1"
```
Comment (m-iwanicki) [inline include/dts-functions.sh:1529]:
```suggestion
          wait_for_network_connection && ${CMD_DASHARO_HCL_REPORT} && LOGS_SENT="1"
```

**Review by m-iwanicki — CHANGES_REQUESTED** [review]
There are couple places where you don't pass `SHELLOPTS`, can you explain why or did you miss it?

* https://github.com/Dasharo/dts-scripts/blob/4b01ce1c81bf845815b9bc5379264f5b118bf712/scripts/dasharo-deploy#L846
* https://github.com/Dasharo/dts-scripts/blob/4b01ce1c81bf845815b9bc5379264f5b118bf712/reports/dasharo-hcl-report#L337
    *  Should probably remove https://github.com/Dasharo/dts-scripts/blob/4b01ce1c81bf845815b9bc5379264f5b118bf712/reports/dasharo-hcl-report#L53 as it's probably unneeded as we source `$DTS_FUNCS`
* https://github.com/Dasharo/dts-scripts/blob/4b01ce1c81bf845815b9bc5379264f5b118bf712/include/dts-functions.sh#L1529

File: include/dts-functions.sh (hunk 1 above)
Comment (PLangowski) [reply include/dts-functions.sh]: https://github.com/Dasharo/dts-scripts/compare/4b01ce1c81bf845815b9bc5379264f5b118bf712..8c3e02f5eefd1c7dab93864c64c75fd18e1d3704

File: include/dts-functions.sh [hunk 2]
```diff
@@ -1715,11 +1724,33 @@ footer_options(){
...
+    log_dir+="_$uuid_$(date +'%Y_%m_%d_%H_%M_%S_%N')"
+    log_dir="${log_dir// /_}"
+    log_dir="${log_dir//\//_}"
+    log_dir="/tmp/${log_dir}"
+
+    mkdir $log_dir
+    cp ${DTS_LOG_FILE} $log_dir
+    cp ${ERR_LOG_FILE} $log_dir
+    if [ -f ${FLASHROM_LOG_FILE} ]; then
+      cp ${FLASHROM_LOG_FILE} $log_dir
```
[the remark this answers is not in the document]
Comment (PLangowski) [reply include/dts-functions.sh:1744]: What do you mean? It's added right above this.

[the remark this answers is not in the document]
Comment (m-iwanicki) [reply include/dts-functions.sh:1744]: You are right, not sure how I missed it.

**Review by m-iwanicki — APPROVED** [review]
Looks good, have you tested it?

**Pull request merged**